common/config/rush/pnpm-lock.yaml

```diff
@@ -6756,7 +6758,7 @@ packages:
     dependencies:
       '@storybook/client-logger': 7.0.0-rc.8
       '@storybook/core-events': 7.0.0-rc.8
-      '@storybook/csf': 0.0.2-next.11
+      '@storybook/csf': 0.1.1-next.0
```

Comment:
The changes happened after running `rush-update` to install the classnames package. The fix works like a charm, will keep the code snippet handy for the other PRs that have similar changes!

Reply:
Nice! glad it worked 👍

Reply:
You can update the packages. I just talked with @eileenmguo to see whether we want to include pre-releases in the installed packages. 
This might be because we identify a prerelease package as the minimal version of the package, signalling we want to have pre-release packages ("^7.0.0-rc.3"). If we change that to not have `-rc.3` it'll only take stable releases.